.github/workflows/test.yml

```
@@ -9,8 +9,27 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.6', '3.7', '3.8', '3.9', 'pypy3']
-        django-version: ['2.2', '3.0', '3.1', '3.2', 'main']
+        python-version: ['3.6', '3.7', '3.8', '3.9', '3.10', 'pypy3']
```

`pypy3` is deprecated, see https://github.com/jazzband/django-pipeline/pull/761. 

```suggestion
        python-version: ['3.6', '3.7', '3.8', '3.9', '3.10', 'pypy-3.8']
```